src/components/OrderBookWidget.tsx

```diff
@@ -1,12 +1,40 @@
 import React, { useEffect, useRef } from 'react'
+import BigNumber from 'bignumber.js'
 import styled from 'styled-components'
-import { TokenDetails, Network } from 'types'
+
 import * as am4core from '@amcharts/amcharts4/core'
 import * as am4charts from '@amcharts/amcharts4/charts'
 import am4themesSpiritedaway from '@amcharts/amcharts4/themes/spiritedaway'
-import { getNetworkFromId, safeTokenName } from '@gnosis.pm/dex-js'
+
 import { dexPriceEstimatorApi } from 'api'
 
+import { getNetworkFromId, safeTokenName, formatSmart, toBN } from 'utils'
+
+import { TEN_BIG_NUMBER } from 'const'
+
+import { TokenDetails, Network } from 'types'
+
+function checkPriceAndConvert(price: number): string {
+  // prices come in as type number so need to be converted
+  // some of the WEI values are so small we need to cast to Wei twice
```

Comment:
What do you mean cast to Wei twice?

Reply:
see my quote reply to Anxo - essentially some WEI values come in super small as well and can't be cast to BN and formatSmart doesn't accept `BigNumber`